fetch-todolist/src/component/Home.js

```diff
@@ -29,29 +46,28 @@ const Home = () => {
   );
 
   const changeStatus = (check, id) => {
-    const updatedStatusArray = data.map((obj) => {
+    const object = data.find((obj) => {
       if (obj.id === id) {
+        console.log("id", id, "check ", check);
```

Comment:
remove logs applicable for whole PR

Reply:
![Screenshot from 2023-02-06 11-34-58](https://user-images.githubusercontent.com/118346581/216895764-23d895ce-7ddd-4840-8c92-3867da136f2e.png)

Reply:
done